src/Illuminate/Database/Query/Grammars/Grammar.php

```diff
@@ -234,7 +234,7 @@ protected function whereBasic(Builder $query, $where)
     {
         $value = $this->parameter($where['value']);
 
-        return $this->wrap($where['column']).' '.$where['operator'].' '.$value;
+        return $this->escapeIfRequired($this->wrap($where['column']).' '.$where['operator'].' '.$value, $where);
```

Comment:
Why would you pass the whole `$where` when you only need `'escape'` within your function?

Would make more sense to use `$where['escape'] ?? null` here (and also not `false`; feels strange; using `null` feels more in-line with how the operator is named)

Reply:
Thanks for your review! I've updated it to be just `escapeIf`, where the second parameter is a boolean. I think this makes sense